src/ReleaseHistory.md

```diff
@@ -13,10 +13,13 @@
 - UER => eliminate unhandled exceptions in rules
 - UEE => eliminate unhandled exceptions in engine
 
-## **v4.0.0-rc5** UNRELEASED
+## **v4.0.1** UNRELEASED
+
+## **v4.0.0**
+* DEP: Update Sarif.Sdk submodule from [120fae3 to 62031c4](https://github.com/microsoft/sarif-sdk/compare/120fae3...62031c4). Full [SARIF SDK Release History](https://github.com/microsoft/sarif-sdk/blob/62031c4/src/ReleaseHistory.md).
 
 ## **v4.0.0-rc4**
-* DEP: Update Sarif.Sdk submodule from [235394a to 120fae35](https://github.com/microsoft/sarif-sdk/compare/235394a...120fae35). Full [SARIF SDK Release History](https://github.com/microsoft/sarif-sdk/blob/120fae35/src/ReleaseHistory.md).
```

Comment:
>120fae35 [](http://example.com/codeflow?start=51&length=8)

I used 8 instead of 7 which was by mistake. It works with either 7 or 8.
a quick try the min working is 5 :)